classes/api/forms.php

```diff
@@ -19,6 +19,31 @@ class Caldera_Forms_API_Forms extends  Caldera_Forms_API_CRUD {
 	 */
 	public function add_routes( $namespace ) {
 		parent::add_routes($namespace);
+        register_rest_route( $namespace, $this->id_endpoint_url(),
+           [
+                'methods'             => \WP_REST_Server::EDITABLE,
+                'callback'            => [ $this, 'save_form' ],
+                'permission_callback' => [ $this, 'save_form_permissions_check' ],
+                'args'                => [
+                    'cf_edit_nonce' => [
+                        'type' => 'string',
+                        'description' => __('Caldera Forms editor nonce'),
+                        'required' => 'true'
+                    ],
+                    'config' => [
+                        'type' => 'object',
+                        'description' => __('Caldera Forms editor nonce'),
```

Comment:
We should add the 'caldera-forms' text-domain to __() function

Reply:
Fixed in b476c474